tests/k6/components/utils.js

```diff
@@ -626,10 +626,12 @@ export function applyModelReplicaChange(config) {
         const modelId = Math.floor(Math.random() * config.maxNumModels[j])
         const modelName = config.modelNamePrefix[j] + modelId.toString()
 
-        let replicas =  Math.floor(Math.random() * config.maxModelReplicas[j]) + 1
-        const model = generateModel(config.modelType[j], modelName, 1, replicas, config.isSchedulerProxy, config.modelMemoryBytes[j], config.inferBatchSize[j])
+        const rand = Math.random()
+        const replicas =  Math.floor(rand * config.maxModelReplicas[j]) + 1
+        const memory = (rand > 0.5)? replicas + "k": "1k"   // this will induce a change in memory causing a new version of the model to be created
```

Comment:
it's a bit awkward to also update the memory in an util function called "applyModelReplicaChange". But let's add it like this, and I can modify things afterwards.

This is because I'm already modifying both replicas and memory in the control-plane tests, and the replica (and now memory) changes in utils.js have been introduced independently. I'll unify those.

Reply:
The memory itself is not the target for the test here, I am just trying to induce a version change in the test, if you are changing it later then perhaps its fine to get it merged for now.